apps/sms/views/conversation/js/conversation.js

```diff
@@ -3000,7 +3001,7 @@ var ConversationView = {
       type: Compose.type
     });
 
-    Drafts.add(draft);
+    Drafts.add(draft).store();
```

Comment:
I'm afraid this would introduce more complexity (one more `if` in `saveDraft`) for not a clear improvement (only saving the line `Drafts.add(draft).store()` ?)